src/wallet/rpcwallet.cpp

```diff
@@ -2804,9 +2804,9 @@ UniValue bumpfee(const JSONRPCRequest& request)
             nNewFeeRate = CWallet::fallbackFee;
         }
 
-        // new fee rate must be at least old rate + minimum relay rate
-        if (nNewFeeRate.GetFeePerK() < nOldFeeRate.GetFeePerK() + ::minRelayTxFee.GetFeePerK()) {
-            nNewFeeRate = CFeeRate(nOldFeeRate.GetFeePerK() + ::minRelayTxFee.GetFeePerK());
+        // new fee rate must be at least old rate + minimum incremental relay rate
+        if (nNewFeeRate.GetFeePerK() < nOldFeeRate.GetFeePerK() + ::incrementalRelayFee.GetFeePerK()) {
```

Comment:
This change (while correct) exposes a bug in the logic, where we weren't otherwise comparing nNewFeeRate to either minTxFee or minRelayTxFee -- we should probably either add a call to compare against GetRequiredFee(), or rewrite the above to just use CWallet::GetMinimumFee.

Separately -- I now realize that we're also not comparing nNewFeeRate to maxTxFee either?  That seems like a separate bug in bumpfee...